packages/snap-preact/src/Templates/SnapTemplates.tsx

```diff
@@ -455,29 +469,74 @@ export function createSnapConfig(templateConfig: SnapTemplatesConfig, templatesS
 	return snapConfig;
 }
 
-function createPlugins(templateConfig: SnapTemplatesConfig, templatesStore: TemplatesStore): PluginGrouping[] {
+function createPlugins(
```

Comment:
I'm not sure that the default values here are necessary (for example scroll to top enabled) - if not, it seems like we could DRY the deepmerge of plugin configs here with some looping or similar... 🤔